packages/opencensus-web-scripts/README.md

```diff
@@ -45,6 +46,28 @@ these two bundles to do instrumentation first and then an export more lazily:
     downloaded and parsed doesn't need to compete with resources for the main
     page load.
 
+If your application already depends on `Zone.js`, the case for `Angular`, use:
+
+* **`./dist/tracing-all-without-zone.js`** - This script combines all the functionality of
+    OC Web, including the `initial-page-load` and `user interaction tracker`. That
```

Comment:
Are the strings "initial-page-load" and "user interaction tracker" meant to refer to code symbols? (That's what the backticks usually mean). If the are just being used for emphasis, consider using * around them to make them italic and removing the dashes in the initial-page-load one.